crates/ruff_python_parser/tests/snapshots/invalid_syntax@class_def_unclosed_type_param_list.py.snap

```diff
@@ -108,19 +108,5 @@ Module(
 1 | class Foo[T1, *T2(a, b):
   |                  ^ Syntax Error: Expected ']', found '('
 2 |     pass
-3 | x = 10
```

Comment:
this is pretty cool! Nice to see how all the work accumulated to now having a single, accurate error message.